eslint/babel-eslint-parser/test/typescript-estree.test.js

```diff
@@ -0,0 +1,136 @@
+import path from "node:path";
+import { parseForESLint } from "../lib/index.cjs";
+import { ESLint } from "eslint";
+import { itDummy, commonJS } from "$repo-utils";
+
+const ESLINT_VERSION = ESLint.version;
+const isESLint9 = ESLINT_VERSION.startsWith("9.");
+const { require } = commonJS(import.meta.url);
+
+const PROPS_TO_REMOVE = [
+  // typescript-eslint generates start/end in the loc object, while Babel generate both start/end and loc
+  { key: "start", type: "Node" },
+  { key: "end", type: "Node" },
+
+  { key: "importKind", type: "Node" },
+  { key: "exportKind", type: "Node" },
+  { key: "variance", type: "Node" },
+  { key: "typeArguments", type: "Node" },
+  { key: "filename", type: null },
+  { key: "identifierName", type: null },
+  // For legacy estree AST
+  { key: "attributes", type: "ImportExpression" },
+];
+
+function deeplyRemoveProperties(obj, props) {
+  for (const [k, v] of Object.entries(obj)) {
+    if (
+      props.some(
+        ({ key, type }) =>
+          key === k &&
+          ((type === "Node" && obj.type) || type === obj.type || type == null),
+      )
+    ) {
+      delete obj[k];
+      continue;
+    }
+
+    if (typeof v === "object") {
+      if (Array.isArray(v)) {
+        for (const el of v) {
+          if (el != null) {
+            deeplyRemoveProperties(el, props);
+          }
+        }
+      }
+
+      if (v != null) {
+        deeplyRemoveProperties(v, props);
+      }
+    }
+  }
+}
+
+(isESLint9 ? describe : describe.skip)(
+  "Babel should output the same AST as TypeScript-Estree",
+  () => {
+    /**
+     * @type {import("@typescript-eslint/typescript-estree")}
+     */
+    let tsEstree;
+
+    /**
+     * @type {import("@typescript-eslint/typescript-estree").TSESTreeOptions}
+     */
+    const tsEstreeOptions = {
+      filePath: "typescript-estree.test.js-test-input.tsx",
+      jsx: true,
+      tokens: true,
+      loc: true,
+      range: true,
+      comment: true,
+      sourceType: "module",
+    };
+
+    function parseAndAssertSame(code, babelEcmaFeatures = null) {
+      if (isESLint9) {
```

Comment:
We seem to have used `describe.skip`?

Reply:
Yeah it was left over from copying the test setup from the eslint integration tests. I will remove it.